playbooks/install.yaml

```diff
@@ -49,8 +49,9 @@
   # we choose raw instead of shell or command as these steps must be run outside the docker container
   - name: install coreos
     become: true
-    raw: coreos-install -d /dev/sda -i ignition.json -C beta
+    raw: coreos-install -d /dev/sda -i ignition.json -C alpha
```

Comment:
ignition 0.31 resolves [an issue where setting file ownership by user name fails](https://github.com/coreos/ignition/issues/722), currently this is only shipping in the alpha release.

This will be parameterized in a future refactoring.